src/components/Popover/Popover.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from "react";
+import {
+  Close,
+  Content,
+  PopoverContentProps,
+  PopoverProps,
+  Portal,
+  Root,
+  Trigger,
+} from "@radix-ui/react-popover";
+import { twMerge } from "tailwind-merge";
+import { IconButton } from "../IconButton";
+
+const Popover = Root;
+
+const PopoverTrigger = Trigger;
+
+interface PopoverHeaderProps {
+  className?: string;
+  title: string;
+}
+const PopoverHeader = ({ className, title }: PopoverHeaderProps) => {
+  return (
+    <div
+      className={twMerge("flex justify-between items-center pb-4", className)}
+    >
+      <p className="text-lg text-text-high-em font-bold">{title}</p>
+      <Close asChild>
+        <IconButton name="cross" size="md" variant="ghost" />
+      </Close>
+    </div>
+  );
+};
+
+export interface PopoverContentWithTitleProps extends PopoverContentProps {
+  title?: string;
+  titleClassName?: string;
+}
+const PopoverContent = React.forwardRef<
+  React.ElementRef<typeof Content>,
+  PopoverContentWithTitleProps
+>(
+  (
+    {
+      align = "center",
+      children,
+      className,
+      sideOffset = 4,
+      title,
+      titleClassName,
+      ...props
+    },
+    ref
+  ) => (
+    <Portal>
+      <Content
+        ref={ref}
+        align={align}
+        sideOffset={sideOffset}
+        className={twMerge(
+          "z-50 min-w-32 rounded-16 border !border-solid border-outline-base-em p-4",
```

Comment:
No need to apply border-solid, specially with !important, it's already the default style 😄 
Let's also apply bg color here. Watch out to pick the right color for figma as it will be the same color name for light and dark mode.


```suggestion
          "z-50 min-w-32 rounded-16 border border-outline-base-em p-4",
```

Reply:
Agree on the bg color, actually moved it to the Popover before checking this comment.

The issue with the border is that if I don't add it this way, it won't display. I'm removing the `!border-solid` just for reference and we can revert it if needed.